extensions/cms/src/cmsUtils.ts

```diff
@@ -24,9 +31,36 @@ const mssqlProvider: string = 'MSSQL';
  */
 export class CmsUtils {
 
+	private _credentialProvider: azdata.CredentialProvider;
 	private _cmsService: mssql.CmsService;
 	private _registeredCmsServers: ICmsResourceNodeInfo[];
 
+	constructor() {
+		// 1) Get a credential provider
+		// 2) Store the credential provider for use later
+		this.setupCredentialProvider();
+	}
+
+	private async setupCredentialProvider(): Promise<boolean> {
+		let credProvider = await azdata.credentials.getProvider(CredentialNamespace);
+		if (credProvider) {
+			this._credentialProvider = credProvider;
+			return true;
+		}
```

Comment:
All code paths need to return a value

Reply:
I still don't see the point in returning anything here - you never use it. 

---
In reply to: [293567713](https://github.com/microsoft/azuredatastudio/pull/5989#discussion_r293567713) [](ancestors = 293567713)